android/build.gradle

```diff
@@ -36,22 +36,21 @@ android {
         abortOnError false
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_11
+        targetCompatibility JavaVersion.VERSION_11
```

Comment:
Is this correct? I believe 1 of them is OpenJDK and the other is Oracle JDK.

Reply:
This is directly from a fresh Capacitor 4 plugin template from `npx @capacitor/create-plugin`, see [here](https://github.com/ionic-team/create-capacitor-plugin/blob/03027bf603626ebfdba084cb9d9a1a359d008554/assets/plugin-template/android/build.gradle#L39-L40). I haven't had any issues when testing however I did need to update my Android Studio to use the updated version mentioned 
![image](https://user-images.githubusercontent.com/26913683/191170858-9ebfec2f-966a-45dd-a6f1-6d4a7323207d.png)

Reply:
I see, no worries.